.github/workflows/centos7.yml

```diff
@@ -71,18 +71,19 @@ jobs:
             CXX: clang++
             IMAGE: centos:7
 # Coverage can only been tested with the GNU compiler
-          - BUILD_MODE: coverage
-            GPG_VERSION: beta
-            RNP_TESTS: cli_tests
-            CC: gcc
-            CXX: g++
-            IMAGE: centos:7
-          - BUILD_MODE: coverage
-            GPG_VERSION: beta
-            RNP_TESTS: rnp_tests
-            CC: gcc
-            CXX: g++
-            IMAGE: centos:7
+# Upload only single report from centos8 to the codecov to avoid mess
```

Comment:
@ni4 should we remove these commented out lines?

Reply:
@ronaldtse Sure, removing. And fixing one more thing :-)